src/main/java/com/snowplowanalytics/client/nsq/Connection.java

```diff
@@ -1,13 +1,13 @@
-package com.github.brainlag.nsq;
-
-import com.github.brainlag.nsq.callbacks.NSQErrorCallback;
-import com.github.brainlag.nsq.exceptions.NSQException;
-import com.github.brainlag.nsq.exceptions.NoConnectionsException;
-import com.github.brainlag.nsq.frames.ErrorFrame;
-import com.github.brainlag.nsq.frames.MessageFrame;
-import com.github.brainlag.nsq.frames.NSQFrame;
-import com.github.brainlag.nsq.frames.ResponseFrame;
-import com.github.brainlag.nsq.netty.NSQClientInitializer;
+package com.snowplowanalytics.client.nsq;
```

Comment:
shouldn't we add an apache 2 headers to the source files? @alexanderdean

Reply:
No, it's MIT licensed (see https://github.com/brainlag/JavaNSQClient), I don't want to get into re-licensing the fork...